alpha/template/composite/builder.go

```diff
@@ -258,7 +278,23 @@ func (cb *CustomBuilder) Build(dir string, td TemplateDefinition) error {
 
 	// custom template should output a valid FBC to STDOUT so we can
 	// build the FBC just like all the other templates.
-	return build(cmd, path.Join(dir, customConfig.Output), cb.builderCfg.OutputType)
+	v, err := cmd.Output()
+	reader := bytes.NewReader(v)
+
+	dcfg, err := declcfg.LoadReader(reader)
+	cmdString := []string{customConfig.Command}
+	cmdString = append(cmdString, customConfig.Args...)
```

Comment:
Nit: I think you could this in one line by doing something like:
```suggestion
	cmdString := append([]string{customConfig.Command}, customConfig.Args...)
```

Reply:
I think this loses in readability what it gains in brevity, but maybe it's just my relative youth w.r.t. golang?